GomokuClient/packages/gomoku-core/src/features/Board/Board.tsx

```diff
@@ -0,0 +1,54 @@
+import styles from "./Board.module.scss";
+import { CellValue } from "@/hooks/useBoard";
+import { useMemo } from "react";
+
+export interface BoardProps {
+  size: number;
+  onTileClick: (x: number, y: number) => void;
+  tiles: CellValue[][];
+}
+
+export const Board = ({ size, onTileClick, tiles }: BoardProps) => {
+  const tilesElements = useMemo(
+    () =>
+      tiles.map((row, rowIndex) =>
+        row.map((col, colIndex) => {
+          return col !== null ? (
+            <div
+              key={`${rowIndex}-${colIndex}`}
+              className={styles.tileContainer}
+            >
+              <div
+                className={`${styles.tile} ${
+                  col === "black" ? styles.tileBlack : styles.tileWhite
+                }`}
+              />
+            </div>
+          ) : (
+            <div
+              key={`${rowIndex}-${colIndex}`}
+              className={styles.tileContainer}
+              onClick={() => onTileClick(rowIndex, colIndex)}
+            />
+          );
+        }),
+      ),
+    [size, tiles, onTileClick],
```

Comment:
useMemo has an unnecessary dependency: 'size'. Either exclude it or remove the dependency array

Reply:
removed from array